src/rpcrawtransaction.cpp

```diff
@@ -529,8 +541,18 @@ Value signrawtransaction(const Array& params, bool fHelp)
 
             "\nResult:\n"
             "{\n"
-            "  \"hex\": \"value\",   (string) The raw transaction with signature(s) (hex-encoded string)\n"
-            "  \"complete\": n       (numeric) if transaction has a complete set of signature (0 if not)\n"
+            "  \"hex\" : \"value\",   (string) The hex-encoded raw transaction with signature(s)\n"
+            "  \"complete\" : b,    (boolean) Whether the transaction has a complete set of signatures\n"
+            "  \"errors\" : [       (json array of objects) Script verification errors (if there are any)\n"
+            "    {\n"
+            "      \"txid\" : \"hash\",       (string) The transaction hash\n"
```

Comment:
Hm.. I'm not sure, if the purpose of this PR was not clearly communicated by me, or if I misread your comment, or if the labels are faulty. The goal is to report any errors related to the signing and spending when using `signrawtransaction` - basically anything that causes a result with `"complete": false`. Reporting which inputs in particular caused failures should help to locate the underlying issues. Same goes for `sequence`: if it can cause errors, it seems relevant in my opinion.